cmd/podman/common/netflags.go

```diff
@@ -26,6 +26,13 @@ func DefineNetFlags(cmd *cobra.Command) {
 	)
 	_ = cmd.RegisterFlagCompletionFunc(addHostFlagName, completion.AutocompleteNone)
 
+	baseHostsFileFlagName := "base-hosts-file"
```

Comment:
I would just call tis hosts-file Make it much easier to find for the users.

Reply:
I am fine with hosts-file as name.